scrapers_next/ma/committees.py

```diff
@@ -0,0 +1,100 @@
+from spatula import URL, HtmlListPage, HtmlPage, CSS, XPath, selectors
+from openstates.models import ScrapeCommittee
+import requests
+
+requests.packages.urllib3.disable_warnings()
+
+
+class CommitteeDetail(HtmlPage):
+    def process_page(self):
+        com = self.input
+        try:
+            for members_area in XPath(
+                '//*[contains(@class, "committeeMemberList")]'
+            ).match_one(self.root)[0]:
+                if "There are no members" in members_area.text:
+                    break
+                # continue to extract members if available
+                for members_info in XPath(
+                    '//*[contains(@class, "committeeMemberList")]'
+                ).match_one(members_area):
+                    for data in members_info:
+                        name = None
+                        role = "Member"
+                        for info in data.text_content().splitlines():
+                            if info.strip():
+                                if name is None:
+                                    name = info.strip()
+                                else:
+                                    role = info.strip()
+                        com.add_member(name, role)
+        except selectors.SelectorError:
+            return com
+        com.add_source(
+            self.source.url,
+            note="Committee Details Page",
+        )
+        return com
+
+
+class CommitteeList(HtmlListPage):
+    source = URL(
+        "https://malegislature.gov/Committees",
+        headers={
```

Comment:
Do we need these headers? I think at least a few of them may cause strange behavior at the edge.

Reply:
we don't need them but I just removed few...